### DISCUSSION TITLE
Preguntas sobre NOTES asociadas con Tasks

### DISCUSSION THREAD

**Discussion by Mauriceac** [Cotalker/documentation#458] [2. Questions and Answers]
Tengo algunas dudas sobre el nuevo feature, Notes (y quizás un pequeño bug).

1. Los "Notes" que son parte de un _task_, no se pueden ver desde la ventana lateral que se abre con la información de la _task_. ¿Esto es normal?
2. 🐞 Desde el _task view_, cuando veo _Details_ de la _task_, a veces no me sale la información para _ver, editar e eliminar_ la Nota (0:34 del video)
3. ¿Se pueden asociar notas ya existentes a una _task_ o siempre deben ser creadas de cero desde el panel de _task_?

Les dejo un video con todo lo anterior (1:39):

https://user-images.githubusercontent.com/47286214/136377307-cabcaf77-e48c-4f42-ae17-344cdd96fd5b.mp4

**Comment by acedward** [comment]
1 & 2. Son efectivamente bugs (creo que están reportados)
3. Se debería poder unir notas existentes con tareas a través de la API. (No existe una forma de hacerlo a través de la UI)

**Reply by acedward** [reply]
Volví a abrir. Creo que 1 y 2 están corregidos.
Pero el 3 deberiamos abir un issue para hacer esto.

**Reply by jdurandpaz** [reply]
@acedward levanto el issue? o ya lo generaste?

**Reply by acedward** [reply]
Si puedes sería bkn. ¿Cómo te tinca que se podrían referenciarse?
